sauce/features/accounts/customFlagNames/settings.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  name: 'CustomFlagNames',
+  type: 'checkbox',
+  default: false,
+  section: 'accounts',
+  title: 'Add Custom Flag Names',
+  description: 'Adds the ability to set custom flag names.'
```

Comment:
Good ideas. I was having issues deciding how much to include. I did have a warning that it was stored locally and would be lost if browser data was cleared, but removed it. Since you brought it up I'll add it back, just reworded and using some markdown.